react/features/conference/components/native/carmode/Conference.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+import { useIsFocused } from '@react-navigation/native';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Text, SafeAreaView, View } from 'react-native';
+import { withSafeAreaInsets } from 'react-native-safe-area-context';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { setAudioOnly } from '../../../../base/audio-only';
+import { Container, LoadingIndicator, TintedView } from '../../../../base/react';
+import { screen } from '../../../../mobile/navigation/routes';
+import ConferenceTimer from '../../ConferenceTimer';
+import { isConnecting } from '../../functions';
+
+import EndMeetingButton from './EndMeetingButton';
+import SoundDeviceButton from './SoundDeviceButton';
+import TitleBar from './TitleBar';
+import { MicrophoneButton } from './microphone-button';
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * Callback on component focused.
+     * Passes the route name to the embedder.
+     */
+     onFocused: Function
+
+}
+
+// eslint-disable-next-line no-empty-function
+const dummy = () => {};
```

Comment:
Is this necessary? 
I think we can escape the linter by adding '?' next to the  _createOnPress TitleBar prop and not make it required, if that is the case.

Reply:
this was some leftover. Removed.